rasa/core/channels/slack.py

```diff
@@ -121,13 +121,19 @@ def name(cls):
     def from_credentials(cls, credentials):
         if not credentials:
             cls.raise_missing_credentials_exception()
-
-        return cls(credentials.get("slack_token"), credentials.get("slack_channel"))
+        return cls(
+            credentials.get("slack_token"),
+            credentials.get("slack_channel"),
+            credentials.get("slack_retry_reason_header", "x-slack-retry-reason"),
+            credentials.get("slack_retry_number_header", "x-slack-retry-num"),
+        )
```

Comment:
Thanks! One last thing — it looks like even though this is listed in the params,
```
errors_ignore_retry: Any error codes given by Slack
                included in this list will be ignored.
                Error codes are listed
                `here <https://api.slack.com/events-api#errors>`_.
```
it doesn’t seem that the initialization of the class actually used that param, it just got set to `None`, because we didn’t pass it here. Can you add `credentials.get("errors_ignore_retry”, None),` here? I think that might help fix our bug 😃

Reply:
oh, that's true. ok I'll add that too

Reply:
thanks! That's it from me